### PR TITLE
[TIMOB-19924]: iOS "onclick" event is not triggered in the WebView if you are listening to multiple "click" events

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-19924

### CODE REVIEW

File: iphone/Classes/TiUIView.m [hunk 1]
```diff
@@ -1046,11 +1046,11 @@ -(UITapGestureRecognizer*)singleTapRecognizer
 		singleTapRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(recognizedTap:)];
 		[self configureGestureRecognizer:singleTapRecognizer];
 		[self addGestureRecognizer:singleTapRecognizer];
-
 		if (doubleTapRecognizer != nil) {
 			[singleTapRecognizer requireGestureRecognizerToFail:doubleTapRecognizer];
 		}
 	}
+    singleTapRecognizer.delegate = self;
```
Comment (hansemannn) [inline iphone/Classes/TiUIView.m]: Indentation here and all other places where you assign this delegate incorrect, please adjust.

**Review by hansemannn — CHANGES_REQUESTED** [review]

File: iphone/Classes/TiUIView.m (hunk 1 above)
Comment (vijaysingh-axway) [inline iphone/Classes/TiUIView.m]: Done.

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
PR approved! Please backport to 6_1_X

**Review by htbryant94 — APPROVED** [review]
Approved! "onclick" event now triggers correctly with multiple click events present. Tested for the added event types implemented from UIGestureRecognizer. 
iOS10.2.11 & iOS9.3

**Pull request merged**